### DISCUSSION TITLE
First pass: Commonhaus Foundation logos

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#45] [Consensus Building]
An initial set of draft logos:

<img src="https://design.jboss.org/commonhaus/images/commonhaus_r1comparison.png" />

Supporting information (each logo with text to the side or below, and at different sizes) is here: https://design.jboss.org/commonhaus/

Note that this is a draft (not a final mark or text mark or font or.. )

State your preferences (they are numbered!).

Let's try a 5 emoji scale: 🚀 👍 ⚖️ 👎 🙅 

Keep comments constructive. Suggestions for variations on a theme, or new ideas not represented here are welcome. 

Background: 

> The general feel I started with, as input, is the concept of a (well-built) tree-house. A place where we come together to work on cool things and support each other that is also connected to live and growing things (the tree). This was an image that resonated with folks when I first floated it (I mean, seriously, as a grown-up, I would still go hang out in a tree-house if I could).
> 
> <img width="500" alt="many tree houses" src="https://github.com/commonhaus/foundation-draft/assets/808713/dbf04db8-5c9c-43f8-9fde-d1ecfa665b70">
> 
> Some of the choices (re: bolder lines) are specifically related to how the image renders at different sizes (see some of the supporting materials). 
> 
> <img width="500" alt="an icon at different sizes" src="https://github.com/commonhaus/foundation-draft/assets/808713/ab3f9f1f-fca3-4d1d-a7b1-179f5e73f92a">
> 
> For larger sizes, other parts of the page, bolder or softer is absolutely something to think about and consider!

**Comment by kenfinnigan** [comment]
1 👎
2 ⚖️
3 🙅 
4 ⚖️
5 🚀
6 ⚖️
7 🚀

**Comment by trustin** [comment]
Can we get some textual explanation and rational behind each logo from the designer? My impression so far:

1 4 7 👍
5 6 ⚖️
2 3 4 👎

One single solid color with black bold lines (or mixture of bold and thin lines) feel 'too strong' to me by the way. Maybe we could design a color logo without black lines, and then make a B/W or gray version based on that?

**Reply by ebullient** [reply]
Moved my comment to the top.

**Comment by ebullient** [comment]
1. 🚀 - this feels like a stained-glass window to me. 
2. ⚖️ - I don't feel strongly for or against this one, but it isn't a favorite.
3. 🙅 - too pointy. Very sleeping-beauty-thorny-hedge as I read it.
4. 👍 - I like the impression of the house rendered by the tree branches. I could go with a lighter green, tbh.
5. 👎 - It's very busy for me.
6. ⚖️ - this one is an interesting flip (the tree inside the house?); feels very evergreen; but I'm not sure there is room enough inside the house for us all to work on cool things.
7. 🚀 - I like this one, too. The C and the H is there, there could be some stylistic things to avoid the standalone 'I'.

**Comment by aalmiray** [comment]
My main driver is getting a logo that looks great on stickers, so ...

🚀 - yes, it looks like a stained glass window. what's up with the narrow strand/branch on the lower left?
🚀 - house contour could be narrower than the trunk's
⚖️ - thorny. 3d house on a flat three. Hmm, either the whole thing is 3d or it isn't.
🚀 - took me a moment to see the house but I love it
👎 - too busy and pointy
👎 - interesting concept, but I rather see the house in the tree, not the other way round.
⚖️ - house looks like a cathedral (a church building with 2 towers) if you squint your eyes. I'm not a religious person but as I was raised in a Catholic household makes me identify these things immediately. Cathedral vs. bazaar vibes. Perhaps taking the "towers" away? But then the H is gone as well.

**Comment by sigrunixia** [comment]
1. 🚀 I am not getting that ititss a house in a tree, BUT, it still has a nature feel and looks "cozy" to me. If a house in a tree is needed, it may need more trunk length, or a conifer-like tree. 
2. 👍 I actually think the house contour is fine. The house needs to be seen at all sizes, and it is very clearly a tree-house. Maybe, some indentations in the trunk for a ladder?
3. 🙅 Dorothy's house flew out of Kansas and landed in a tree, and not on a witch. 
4. ⚖️ Feels very generic and government-like. It needs some fruit, like.. pomegranate or apples. What fruits do people share most commonly?
5. 🙅 Any edits I would make to this would fundamentally change it, so saying no. It's too busy. The house not defined. 
6. 👍 I like it. Crazy idea.. what if the tree was going out of the house? The house is the seed. 
7. 🚀 I do like this because it is clean, but I would rather have number 1. We need more colorful logos, especially green logos, that are not about climate initiatives or finances. I think this logo would be great for paperwork.

**Comment by cealsair** [comment]
2 🚀 I like this one the most. It has color (a bit more exciting than monochrome), roots at the feet (grassroots movement comes to mind), and reminiscent of a 5-leaf clover (symbol of good fortune)
7 👍 I like this one as a second option. Although it has no color (just black), I like its simplicity. I think making the "haus" more prominent so it looks less like an H or an N would be good. Also, it looks a bit like a church, which brings to mind "grassroots movement, congregation of like-minds, following, etc. However, since contributors would be from all over the world, we may want to make it look less like a church?

1 and 5. ⚖️ It's nice. It looks a bit like a church stained window.
4 and 6. 👎
3 🙅 It looks like it has thorns so "don't touch it"

**Comment by jkschneider** [comment]
I like many of them quite a bit. 

Favorites: 1, 2, 4.
Runners up: 5, 6
Not as favored: 3, 7

**Comment by cowtowncoder** [comment]
For me it is difficult to give strict preference, but I note one thing that I like -- I like icons with 3 colors (green, white, black) which means I don't like 4 and 7 quite as much. Similarly use of outlines is pleasing to my eye, so 1, 3 and maybe 2 are slightly preferable over the rest.

Son in some ways I would love to see sort of merger of 1 and 3, with little less "busy" than 1, and without spikes 3 has (I guess those are branches but they somehow look like spikes on first glance). And while I like some things in 2, it's conversely bit too plain.

So of existing options as is, my favorites are 1 and 3.

**Comment by ebullient** [comment]
Closing the first pass review. Looking forward to round 2!
